tox.ini

```diff
@@ -19,42 +20,48 @@ basepython =
 commands =
     /bin/sh -c '\
         export DJANGO_SETTINGS_MODULE=readthedocs.settings.test; \
-        pytest --cov-report= --cov-config {toxinidir}/.coveragerc --cov=. --suppress-no-test-exit-code -m "not proxito" {posargs:{env:TOX_POSARGS:-m "not search and not proxito"}}'
+        pytest --cov-report= --cov-config {toxinidir}/.coveragerc --cov=. -m "not proxito" {posargs}'
 
+[testenv:proxito]
+description = Run test suite for proxito with {basepython}.
+commands =
     /bin/sh -c '\
         export DJANGO_SETTINGS_MODULE=readthedocs.settings.proxito.test; \
-        pytest --cov-report= --cov-config {toxinidir}/.coveragerc --cov=. --cov-append -m proxito --suppress-no-test-exit-code {posargs}'
+        pytest --cov-report= --cov-config {toxinidir}/.coveragerc --cov=. --cov-append -m proxito {posargs}'
 
+[testenv:migrations]
+description = check for missing migrations
+changedir = {toxinidir}
+commands =
+    ./manage.py makemigrations --check --dry-run
 
 [testenv:docs]
 description = build readthedocs documentation
+envdir = {toxinidir}/.docs
 changedir = {toxinidir}/docs
 commands =
     pip install -r{toxinidir}/requirements/docs.txt  # TODO: Remove this once our Sphinx versions don't conflict
     sphinx-build -W --keep-going -b html -d {envtmpdir}/doctrees . {envtmpdir}/html
 
 [testenv:docs-linkcheck]
 description = Check for broken links in the docs
+envdir = {toxinidir}/.docs-linkcheck
 changedir = {toxinidir}/docs
 commands =
     pip install -r{toxinidir}/requirements/docs.txt  # TODO: Remove this once our Sphinx versions don't conflict
     sphinx-build -W --keep-going -q -b linkcheck -d {envtmpdir}/doctrees . {envtmpdir}/linkcheck
 
-[testenv:migrations]
-description = check for missing migrations
-changedir = {toxinidir}
-commands =
-    ./manage.py makemigrations --check --dry-run
-
 [testenv:docs-lint]
 description = run linter (rstcheck) to ensure there aren't errors on our docs
+envdir = {toxinidir}/.docs-lint
```

Comment:
Why do we need to make it explicit for all the testenv now? Isn't easier to just specify `proxito` and `py36` as equal instead?

Reply:
the override comes from the main environment "py36", all other environments inherit from this one.